### DISCUSSION TITLE
[v6] How to delete an uploaded file with adonisjs please ?

### DISCUSSION THREAD

**Discussion by franckDev21** [adonisjs/core#4536] [Help] [answered]
I'd like to know how to delete the files I previously downloaded in a specific folder. Please, how can I do that?

**Comment by MaximeMRF** [comment, marked as answer]
Hello @franckDev21, you can stock in a sql table the path of your file.
For example if you have a Profile table you create one column named `picture_link` that is the link to your image on your upload folder. Then if you want to delete the image or file you have just to retrieve the path with the `picture_link` column then delete it with the filesystem.

**Reply by franckDev21** [reply]
Right, thanks, with an fs.unlink(), isn't it ?

**Reply by MaximeMRF** [reply]
Yess, exactly

**Reply by franckDev21** [reply]
😊😊 Thank you
